src/lib/search/index.ts

```diff
@@ -13,7 +13,8 @@ import {
 } from "../util/validate/validate";
 
 interface ISearchObject {
-  searchTerm: string
+  searchTerm: string,
+  apiKey: string,
```

Comment:
In TS if the value is optional, it's followed by `?` - this is just compile-time checking, but I'll add tests and this may break them. Can you put that in? Otherwise LGTM!

Reply:
Done!